website/sidebars.json

```diff
@@ -30,32 +30,24 @@
       "exception",
       "object"
     ],
-    "JavaScript": [
+    "JavaScript (obsolete)": [
```

Comment:
I don't like the idea of adding "(obsolete)" here. Either they are actually obsolete and we should remove the links entirely, or they are not obsolete. Is there a reason this docs should stick around? Can we just remove this section and add disclaimers to the old markdown files?

Reply:
Yeah, might be too confusing. I removed the `(obsolete)` addition for now. Ideally there would be a little JavaScript introduction that is JSOO centric.

Ideally I'd like to remove the sidebar section completely, but I am not sure how many users are still cross-referencing that page. Maybe at a later point in time we should completely remove those pages all together I guess.